github/repos_projects_test.go

```diff
@@ -91,9 +95,10 @@ func TestRepositoriesService_UpdateProject(t *testing.T) {
 	setup()
 	defer teardown()
 
+	updateUrl := fmt.Sprintf("%s/1", projectsUrl)
```

Comment:
Inline this, no need for constants or fmt.Sprintf, just write the test value directly as a string. It's more readable that way.